check_coverage.sh

```diff
@@ -21,9 +21,9 @@ go_tool_cover_output=$(go tool cover -func=coverage.out)
 
 echo "$go_tool_cover_output"
 
-if (( $(echo $go_tool_cover_output | grep -i total | awk '{print $NF}' | egrep "^[-0-9]+" -o) > $THRESHOLD )); then
+if (( $(echo "$go_tool_cover_output" | grep -i total | awk '{print $NF}' | grep -E "^[-0-9]+" -o) > THRESHOLD )); then
```

Comment:
also this `"^[-0-9]+"` should be `"^[0-9]+"`

Reply:
Yeh, -55% is not proper coverage :)